src/phoenix/trace/utils.py

```diff
@@ -1,11 +1,34 @@
 import json
+import os
 import re
 from traceback import format_exception
 from typing import List, Optional, Tuple, cast
+from urllib import request
 
 import pandas as pd
 
 
+def is_jsonl_file(file_path: str) -> bool:
+    """
+    Check if the given file is a Parquet file.
+    """
+    file_extension = os.path.splitext(file_path)[-1]
+    if file_extension == ".jsonl":
+        return True
+    return False
+
+
+def store_json_traces_fixture(
```

Comment:
```suggestion
def download_json_traces_fixture(
```

"store" suggests this function persists the data somehow.